src/edit/DistortHandle.js

```diff
@@ -13,8 +13,10 @@ L.DistortHandle = L.EditHandle.extend({
 	},
 
 	_onHandleDrag: function() {
+		window.corner = this._corner;
```

Comment:
Are we using this anywhere?

Reply:
good catch! This one, if you are curious, is actually what I used to figure out that our corners are in a weird order. The topleft corner (according to Leaflet documentation and our own repo code) is the origin and should be corner 0. But we make it initially corner 1, and the corner to the right of it 0. If we turn the image inside out, the corners swap. 

Perhaps I will just comment this out and write a little comment above it, "uncomment this if you need to see which corner # you are dragging"?

Reply:
Sure! That'll be fine, or perhaps maybe we can indicate the users to use the `window` object for this purpose in the `README.md` (hence keeping things cleaner in the code)? But I'm okay with whatever approach you see fit!

Reply:
@rexagod ok as commented below on the _hidetoolbar / _showtoolbar this is also updated with comment. 

OHH readme idea is actually smart. You know what I will do for this one, I will write a `getCorner` method and it to the README section about corners. I will also update this._corners to use getCorners because _corners should be the private one and the methods should be part of the exposed API

Reply:
Sounds good to me! 👍